PlantFriends.ino

```diff
@@ -212,13 +234,15 @@ static void ICACHE_RAM_ATTR soilInterrupt() {
 
 static bool readSoil(uint32_t *soil_measurements, size_t n_meas) {
   pinMode(SOIL_OUT, OUTPUT);
-  pinMode(SOIL_IN, INPUT);
 
   bool result = true;
   for (size_t i = 0; i < n_meas; i++) {
     Serial.print(i); Serial.print(": ");
     digitalWrite(SOIL_OUT, LOW);
+    pinMode(SOIL_IN, OUTPUT);
+    digitalWrite(SOIL_IN, LOW);
     delay(10); // Wait for the voltage to drop
+    pinMode(SOIL_IN, INPUT);
```

Comment:
Add comment on why we should actively drive `SOIL_IN` low, as we talked about. Something like this:

> The ESP8266 can sink 12 mA max, at 4.2 V this equals 350 ohm. At 1 uF, which is very high this means that the capacitor time constant is 350 us. By waiting 5 times the time constant 1750 us), the voltage can be assumed to be 0 V. In conclusion waiting 10 ms should be plenty. Check out: http://www.learningaboutelectronics.com/Articles/How-long-does-it-take-to-discharge-a-capacitor for more information